manim/mobject/mobject.py

```diff
@@ -54,7 +54,7 @@ class Mobject(Container):
     """
 
     def __init__(self, color=WHITE, name=None, dim=3, target=None, z_index=0, **kwargs):
-        self.color = Color(color)
+        self.color: Color = Color(color)
```

Comment:
The typing hints should be done in the parameters one line above -- not in the definition/code

Reply:
okay, I will remove this one then.
```suggestion
        self.color  = Color(color)
```